app/styles/base.css

```diff
@@ -30,30 +30,30 @@
  */
 
 :root {
-  font-family: sans-serif;
+    font-family: sans-serif;
```

Comment:
Is there a CSS lint we can turn on to avoid issues in the future?

Reply:
You can tell GitHub to ignore specified commits in git blame:
https://docs.github.com/en/repositories/working-with-files/using-files/viewing-and-understanding-files#ignore-commits-in-the-blame-view

I use prettier to format html/css/js/markdown but you could configure it for CSS only: https://prettier.io/

Reply:
Oooh! Nice! I was not aware of that feature. Might be a bunch of old commits we want to add to that as well.